src/os/command/feature/featurefillcolorcmd.js

```diff
@@ -0,0 +1,124 @@
+goog.provide('os.command.FeatureFillColor');
+
+goog.require('os.command.AbstractFeatureStyle');
+goog.require('os.events.PropertyChangeEvent');
+goog.require('os.metrics');
+
+
+
+/**
+ * Changes the fill color of a feature
+ * @extends {os.command.AbstractFeatureStyle}
+ * @param {string} layerId
+ * @param {string} featureId
+ * @param {Array<number>|string} color
+ * @param {(Array<number>|string)=} opt_oldColor
+ * @constructor
+ */
+os.command.FeatureFillColor = function(layerId, featureId, color, opt_oldColor) {
+  os.command.FeatureFillColor.base(this, 'constructor', layerId, featureId, color, opt_oldColor);
+  this.title = 'Change Feature Fill Color';
+  this.metricKey = os.metrics.Layer.FEATURE_FILL_COLOR;
+
+  if (!color) {
+    var feature = /** @type {ol.Feature} */ (this.getFeature());
```

Comment:
This typecast should not be necessary. That function is already typed to return an `ol.Feature`, so this cast doesn't tell the compiler anything it doesn't already know.

Reply:
This should now be fixed. I will squash commits at the end to add the breaking change message.